docs/exposing-binding-data/direct-secret-reference.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 3
+---
+
+# Direct Secret Reference
+
+[Direct Secret Reference][direct-secret-reference] allows to connect to a
+service by using Secret as a reference.  This is useful when there is no
+Provisioned Service available.
```

Comment:
did we introduce anywhere before the notion of Provisioned Service?

We should explain what binding to a secret really means - are all keys exposed, or?